packages/st2/debian/st2.links

```diff
@@ -2,5 +2,6 @@ opt/stackstorm/st2/bin/st2 usr/bin/st2
 opt/stackstorm/st2/bin/st2-trigger-refire usr/bin/st2-trigger-refire
 opt/stackstorm/st2/bin/st2-rule-tester usr/bin/st2-rule-tester
 opt/stackstorm/st2/bin/st2-register-content usr/bin/st2-register-content
+opt/stackstorm/st2/bin/st2-apply-rbac-definitions usr/bin/st2-apply-rbac-definitions
```

Comment:
Answering my own question - it looks like there is some magic happening in `rpmspec/helpers.spec` and `debian/st2.links` it also used for RHEL packages.